### PR TITLE
blog: new feature for treehouses cli

### PR DESCRIPTION
**Pull request by irisb1701**
## Description
Step by step instructions for new interns on how to create a new feature for the cli

##Link

https://raw.githack.com/treehouses/treehouses.github.io/blog_feature/#!pages/blog/20190831-newfeature.md

### CODE REVIEW

File: pages/blog/20190831-newfeature.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+Looking at the treehouses cli commands, you notice there aren't any commands involving detecting temperature and decide it would be a useful addition.
+
+Now that you have an idea for a new command, you will create a new issue and when approved, can create a new branch and begin working on it.
+
+
+## Step 0 - Vim
+
+Vim is an easy way to test your code. Here are instructions on how to use it.
+
+1. Download Vim  (or type `brew install vim` in your terminal)
```
Comment (Liew211) [inline pages/blog/20190831-newfeature.md]: Add link to Vim - https://www.vim.org/download.php

Maybe also differentiate between Windows and macOS?

**Comment by dogi** [comment]
@irisb1701 picture naming not blog conform ;)

**Comment by vmnet8** [comment]
Looks perfect to me.

**Review by vmnet8 — APPROVED** [review]
look perfect to me.

File: pages/blog/20190831-newfeature.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+1. Inside the function we will call `vcgencmd measure_temp` by simply writing in the command
+1. ![](../../images/vgenc.png)
+1. This can be the end of the function, however we want to modify what is returned
+    1. First lets put the `vcgencmd measure_temp` command in a variable and call it "reading" so we can refer to it: `reading=$(vcgencmd measure_temp)`
+    1. Lets make a new variable called number0 referring `reading` to remove "temp=": `number0=${reading:5}`
+    1. Now let's make a third variable callled number which refers to number0 to remove `'C`: `number=${number0/%??/}`
+    1. Now if we `echo $number` we will get the pure number.
+    1. We're going to finish the function by using `echo $number"°C"`
+    1. This returns our desired output which may look something like 65.34°C
+    1. ![](../../images/no_case.png)
```
Comment (phamduchongan93) [inline pages/blog/20190831-newfeature.md]: Picture wasn't found

**Review by phamduchongan93 — COMMENTED** [review]
Need to redirect the link

**Comment by Kid243** [comment]
On Step 4, you have an extra bullet where it's empty. But other than that, I believe the blog is basically ready
IrisBlogReview1

**Review by Kid243 — APPROVED** [review]

File: pages/blog/20190831-newfeature.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+1. ![](../../pages/blog/images/20190831-vim-commands.png)
+
+
+## Step 1: Creating your new Command
+Your function can be as simple as just calling the `vcgencmd measure_temp` command, however this command will return something like "temp=60.00'C".
+You may want to modify it.
+
+1. Your first line will define a function which we will call temperature
+1. Inside the function we will call `vcgencmd measure_temp` by simply writing in the command
+1. ![](../../pages/blog/images/20190831-vgenc.png)
```
Comment (CalebProvost) [inline pages/blog/20190831-newfeature.md]: @Kid243 the floating list number is due to an error in the image that's suppose to show up their. As @phamduchongan93 pointed out, the error is due to the path. I believe it should be ``

Looks great @irisb1701. Excellent job on the tutorial/blog.

**Review by CalebProvost — APPROVED** [review]

**Review by dogi — APPROVED** [review]

**Pull request merged**